frontend/src/pages/forms/Form.jsx

```diff
@@ -13,6 +13,7 @@ import CongratulationsModal from 'components/congratulationsModal/Congratulation
 import StyledPopup from 'components/openpositionpopup/StyledPopup';
 import { useQuery } from '@tanstack/react-query';
 import axios from 'axios';
+import Button from '../../components/Button/Button';
```

Comment:
please us absolute imports here and in other places, so after putting it into new folder it should be 
`import { Button } from "components/ui/Button"`